client/my-sites/plans/ecommerce-trial/trial-feature-card/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card, Gridicon } from '@automattic/components';
+import { useState } from 'react';
+
+import './style.scss';
+
+interface Props {
```

Comment:
Nit: I would prefer this to be called `TrialFeatureCardProps` so the intent is super-explicit.